SAMLPortal/Models/GlobalSettings.cs

```diff
@@ -1,7 +1,9 @@
+using System.Security.Principal;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Provide an 'AssemblyVersion' attribute for this assembly.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)